dyn_sim/ctrl/segway/fdbk_lin_segway_ctrl.py

```diff
@@ -0,0 +1,109 @@
+import numpy as np
+
+from dyn_sim.ctrl.ctrl import Controller
+from dyn_sim.sys.planar.segway import Segway
+
+# constants
+g = 9.80665  # gravitational acceleration
+
+
+class FLPosSegwayController(Controller):
+    """Feedback linearizing controller for segway.
+
+    Only capable of position regulation at the moment.
+    Could implement trajectory following using Callables.
```

Comment:
don't mention traj following - the name of the controller suggests position regulation already (maybe add "Reg" as part of the class name). in the docstring's first line, mention position regulation.

Reply:
also, i think "input-output linearization" (i.e. IOL) is a bit more descriptive of the setting. i personally feel that saying "feedback linearization" implies full-state feedback or at least muddies the waters, whereas "input-output linearization" makes it clear we're only controlling on outputs. not sure if you agree, this is more of a philosophical thing though

Reply:
I see that but in context of CDS233 we've called these 'feedback linearizing'. I will leave as is for now, but a potential style change for the future if you reuse this code for another project